pkgs/games/dwarf-fortress/twbt/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenvNoCC, fetchurl, unzip }:
```

Comment:
This is kind of a nitpick but I prefer to just use stdenv & put ```stdenv = stdenvNoCC;```.

Reply:
Whether or not it needs the compiler is a property of the TWBT package, no? Not its caller. There is no use in overriding that parameter.

Reply:
Oh yes that's make sense. Would it be difficult to make it build from source? They provide a CMakeFiles.txt that looks pretty simple.

Reply:
They haven't tagged their releases, so I wouldn't know which git reference to use. I don't think we'd gain anything from compiling from source, either?

Reply:
It would make it more portable. For instance "twbt.plug.so" is compiled for x86_64-linux. Someone might want to compile a version for i686-linux as well - or something besides linux. We also frequently hit issues where the linked libc.so or libgcc.so are not the same version on the system so it hits an issue (especially on NixOS where things are not always in the right place).

Reply:
All right, but we still don't have release tags. I'll raise the issue with the author; in the meantime, I'll edit this commit to include 32-bit linux as an option. Kind of forgot that people still use that.

Reply:
Ok yeah I would say for now it's okay - just definitely possible to improve it later on.